scalafmt-tests/src/test/resources/test/ContinuationIndent.stat

```diff
@@ -37,3 +37,15 @@ class ExtendTest
     with D
     with E =>
 }
+<<< #1334 1
```

Comment:
it'll be helpful if you name the test with more meaningful description (you can include #1334 to point the issue number of course :) )

Reply:
that was an oversight, thanks for noticing.